connections.js

```diff
@@ -0,0 +1,64 @@
+import { MongoClient } from 'mongodb';
+import functions from './database.js';
```

Comment:
```
import { addNewSoldier, lookForSoldier, lookForAllSoldiers } from './database.js';
```

Reply:
משום מה זה לא אוהב את זה. eslint מקפית הודעה כזו על כל פונקציה:
lookForSoldier not found in './database.js' 
למרות שהכן כן שם אבל כשאני שמה את זה במשתנה ועושה לו import זה עובד

Reply:
It is because you `export default` in `database.js` instead of `export`. use default only when you are exporting one thing

Reply:
fixed